media-libs/Field3D/files/Field3D-1.7.2-openexr-fixes.patch

```diff
@@ -0,0 +1,38 @@
+diff -purN a/cmake/FindILMBase.cmake b/cmake/FindILMBase.cmake
+--- a/cmake/FindILMBase.cmake	2016-06-19 05:56:00.000000000 +0100
++++ b/cmake/FindILMBase.cmake	2017-03-02 11:44:35.449043588 +0000
+@@ -37,12 +37,17 @@
+ #  ILMBASE_FOUND - true if ILMBASE was found on the system
+ #  ILMBASE_LIBRARY_DIRS - the full set of library directories
+ 
++find_package(PkgConfig QUIET)
++if(PKG_CONFIG_FOUND)
++  pkg_check_modules(PC_ILMBASE QUIET IlmBase)
++endif()
```

Comment:
I have no clue. This was an upstream patch. I can change it later and submit something decent upstream.